Updates for SDK X 10.3.0 (#107)
helpshift/helpshift-docs: docs/sdkx_android/getting-started.mdx

If you are on SDK X or migrating to SDK X please note the following requirements:

> * SDK X is functional only above API Level 21 i.e Android OS 5 Lollipop and above.

Reviewer: ```suggestion
* SDK X is functional only above API Level 24 i.e Android OS 7 Nougat and above.
```

* The `minSDKVersion` in code is kept as API Level 16 to allow importing the SDK and compiling in apps that allow installation till API Level 16 i.e Android 4.4 Jelly Bean.

> * If your app is installed on a device below API level 21 (i.e below Android OS 5 Lollipop) then you can choose to have a different medium for providing support. For example, Email, Web etc. We throw an `UnsupportedOSVersionException` if the SDK is initialized below API Level 21. Please catch and use this exception according to your needs. Refer: [Initializing the SDK](/sdkx_android/getting-started/#initializing).

Reviewer: ```suggestion
* If your app is installed on a device below API level 24 (i.e below Android OS 7 Nougat) then you can choose to have a different medium for providing support. For example, Email, Web etc. We throw an `UnsupportedOSVersionException` if the SDK is initialized below API Level 24. Please catch and use this exception according to your needs. Refer: [Initializing the SDK](/sdkx_android/getting-started/#initializing).
```

<Admonition type="info" title="Note">

>   Update (August 2023): SDK X 10.2.x and above is functional only above API Level 24 i.e Android OS 7 Nougat and above

Reviewer 2: Shouldn’t it be 
Update (October 2023)

Reviewer 3: For 10.2.X it was already August.

Reviewer 2: Okay

</Admonition>

### Gradle based projects {#gradle-support}

Add the following dependencies to your `build.gradle` file inside the dependencies section.

<Tabs groupId="android-dependencies-example">

<TabItem value=">= 10.3.0" label="10.3.X">

```groovy
    dependencies {
        implementation 'androidx.appcompat:appcompat:1.0.0'
        implementation 'com.helpshift:helpshift-sdkx:[% sdkx_android.sdk_version  %]'
    }
```

</TabItem>

> <TabItem value="<= 10.2.X>" label="10.2.X">

Reviewer: ```suggestion
<TabItem value="<= 10.2.X>" label="Below 10.2.2">
```

If you are on SDK X or migrating to SDK X please note the following requirements:

* SDK X is functional only above API Level 24 i.e Android OS 7 Nougat and above.
* The `minSDKVersion` in code is kept as API Level 16 to allow importing the SDK and compiling in apps that allow installation till API Level 16 i.e Android 4.4 Jelly Bean.
* If your app is installed on a device below API level 24 (i.e below Android OS 7 Nougat) then you can choose to have a different medium for providing support. For example, Email, Web etc. We throw an `UnsupportedOSVersionException` if the SDK is initialized below API Level 24. Please catch and use this exception according to your needs. Refer: [Initializing the SDK](/sdkx_android/getting-started/#initializing).

> * Please [**refer here**](https://support.helpshift.com/hc/en/13-helpshift-technical-support/faq/1217-sdk-x-os-technical-support-policy/) for our Technical OS Support Policy

Reviewer 2: Please [**refer here**](https://support.helpshift.com/hc/en/13-helpshift-technical-support/faq/1217-sdk-x-os-technical-support-policy/) for our Technical OS Support Policy.

> * Please [**refer here**](https://support.helpshift.com/hc/en/13-helpshift-technical-support/faq/1217-sdk-x-os-technical-support-policy/) for our Technical OS Support Policy

Reviewer 2: ```suggestion
* Please [**refer here**](https://support.helpshift.com/hc/en/13-helpshift-technical-support/faq/1217-sdk-x-os-technical-support-policy/) for our Technical OS Support Policy.
```

<Admonition type="info" title="Note">

>   Update (August 2023): SDK X 10.2.x and above is functional only above API Level 24 i.e Android OS 7 Nougat and above 

Reviewer 2: ```suggestion
  Update (August 2023): SDK X 10.2.x and above is functional only above API Level 24 i.e Android OS 7 Nougat and above.
```

>   Update (August 2023): SDK X 10.2.x and above is functional only above API Level 24 i.e Android OS 7 Nougat and above 

Reviewer 2: ```suggestion
  Update (August 2023): SDK X 10.2.x and above is functional only above API Level 24 i.e Android OS 7 Nougat and above.
```
